### DISCUSSION TITLE
Entry pages

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#226] [Anything Else]
"The only thing what I miss that we can't check which page was the first what the user have opened in a session."

request from [Twitter](https://twitter.com/f3r4lh34rt/status/1283400173335584768)

And we already have request #101 for Exit pages.

**Comment by metmarkosaric** [comment]
You can now click on any referral source for a referral drill-down. See the traffic sent to you by that referral source and the entry pages too.

**Reply by metmarkosaric** [reply]
both entry and exit pages are now live

**Reply by michael-v-bloom** [reply]
Just to be sure, do the entry and exit pages features are also available on the hosted version of Plausible? Thanks.

**Reply by metmarkosaric** [reply]
yes. you can see the live demo of our hosted version here and it includes entry and exit pages: https://plausible.io/plausible.io 

the self-hosted version is a [long term release](https://plausible.io/blog/building-open-source) and is updated on a less frequent schedule. features are released on the hosted version first before moving down to self-hosted

**Reply by michael-v-bloom** [reply]
Thank you for the clarification. I was referring to the self-hosted version, and you've answered my question. Looking forward to the next release then. Best regards.

**Reply by metmarkosaric** [reply]
you're welcome!
